### PR TITLE
Holidays for Georgia(Country)

### PR DESCRIPTION
**Pull request by scor1on**

### CODE REVIEW

**Review by gerryster — CHANGES_REQUESTED** [review]
Thanks for your contribution! I added one minor comment regarding attribution. Feel free to add yourself to the contributors section of package.json. I am going to assign this PR to @commenthol in case he wants to take a look.

**Review by commenthol — APPROVED** [review]

**Comment by commenthol** [comment]
@scor1on Thanks for your great contribution. Please update the attribution then we are ready to merge.

File: data/countries/GE.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+holidays:
+  # @source https://en.wikipedia.org/wiki/Public_holidays_in_Georgia_(country)
```
[the remark this answers is not in the document]
Comment (scor1on) [reply data/countries/GE.yaml]: Done

**Comment by commenthol** [comment]
@scor1on Thanks for you contribution

**Pull request merged**